src/pods/toolbar/components/export-button/export-button.business.spec.ts

```diff
@@ -1948,3 +1953,165 @@ describe('export-button.business', () => {
+
+describe('getPropertyJsonSchema', () => {
+  it('should generate json schema for non-array field', () => {
+    //Arrange
+    const field: FieldVm = {
+      id: '1',
+      PK: false,
+      name: 'fieldName',
+      type: 'string',
+    };
+
+    //Act
+    const result = getPropertyJsonSchema(field);
+
+    //Assert
+    expect(result).toBe('"fieldName": { bsonType: "string" }');
+  });
+
+  it('should generate json schema for array field', () => {
+    //Arrange
+    const field: FieldVm = {
+      id: 'some-id',
+      PK: false,
+      name: 'fieldName',
+      type: 'string',
+      isArray: true,
+    };
+
+    //Act
+    const result = getPropertyJsonSchema(field);
+
+    //Assert
+    expect(result).toBe(
+      '"fieldName": { bsonType: "array", items: { bsonType: "string" } }'
+    );
+  });
+});
+
+describe('getPropertiesJsonSchema', () => {
+  it('should generate json schema for multiple fields', () => {
+    //Arrange
+    const fields: FieldVm[] = [
+      {
+        id: 'id1',
+        PK: false,
+        name: 'field1',
+        type: 'string',
+      },
+      {
+        id: 'id2',
+        PK: false,
+        name: 'field2',
+        type: 'string',
+      },
+    ];
+
+    //Act
+    const result = getPropertiesJsonSchema(fields);
+
+    //Assert
+    expect(result).toEqual(
+      '"field1": { bsonType: "string" },\n        "field2": { bsonType: "string" }'
+    );
+  });
+});
+
+describe('getRequiredFields', () => {
+  it('shoul be return', () => {
+    //Arrange
+    const fields: FieldVm[] = [
+      { id: 'id1', PK: false, name: 'field1', type: 'string' },
+      { id: 'id2', PK: false, name: 'field2', type: 'enum' },
+    ];
+
+    //Act
+    const result = getRequiredFields(fields);
+
+    //Assert
+    expect(result).toBe('');
+  });
+});
+
+describe('getSchemaScriptFromTableVm', () => {
+  it('should generate schema script for a table with fields', () => {
+    //Arrange
+    const table: TableVm = {
+      id: 'tableId',
+      x: 10,
+      y: 20,
+      tableName: 'Table',
+      fields: [
+        { id: 'id1', PK: false, name: 'field1', type: 'string', isNN: true },
+        { id: 'id2', PK: false, name: 'field2', type: 'decimal', isNN: false },
+        { id: 'id3', PK: false, name: 'field3', type: 'bool', isNN: true },
+      ],
+    };
+    //Act
+    const result = getSchemaScriptFromTableVm(table);
+
+    //Assert
+    const expectedScript = `db.createCollection("Table", {
+  validator: {
+    $jsonSchema: {
+      bsonType: "object",
+      title: "Table",
+      required: ["field1", "field3"],
+      properties: {
+        "field1": { bsonType: "string" },
+        "field2": { bsonType: "decimal" },
+        "field3": { bsonType: "bool" },
+      },
+    },
+  },
+});`;
+
+    expect(result).toEqual(expectedScript);
+  });
+});
+
+describe('getSchemaScriptFromTableVmArray', () => {
+  it('should generate schema scripts for an array of tables', () => {
+    //Arrange
+    const tables: TableVm[] = [
```

Comment:
add more than one collection/table, if not this test is the same as the previous one